src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.tsx

```diff
@@ -155,10 +155,9 @@ const TranscriptsFilter = (props: Props) => {
       <Checkbox
         key={key}
         classNames={{
-          unchecked: styles.checkboxUnchecked,
-          checked: styles.checkboxChecked
+          default: styles.checkboxDefault
```

Comment:
Does the `styles.checkboxDefault` class actually exist? What does it do?

Reply:
Yes it does for adding a margin top: https://github.com/Ensembl/ensembl-client/blob/57af4751ad6bf3ba8325fe3b133281da7175c758/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss#L50

Reply:
✅